buildSrc/src/main/groovy/com/carrotsearch/gradle/junit4/RandomizedTestingTask.groovy

```diff
@@ -95,6 +97,17 @@ class RandomizedTestingTask extends DefaultTask {
         listenersConfig.listeners.add(new TestReportLogger(logger: logger, config: testLoggingConfig))
     }
 
+    void basedOn(RandomizedTestingTask other) {
+        configure(BuildPlugin.commonTestConfig(project))
```

Comment:
BuildPlugin should not be referred to here. The intention has always been to move the randomized testing gradle code (notice this is in a different java package than the rest of the buildSrc code) back upstream with randomized testing itself. See https://github.com/randomizedtesting/gradle-randomized-testing-plugin where I made some snapshots when first working on migrating the build to gradle, but was not able to cleanup and document it enough to release on its own.

Reply:
I  moved it back to the Gradle script.